_sass/_navigation.scss

```diff
@@ -0,0 +1,56 @@
+nav {
+    margin-top: 10px;
+}
+
+nav ul {
+    list-style: none;
+    float: left;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+}
+
+nav ul a {
+    box-shadow: none;
+    text-shadow: none;
+    display: block;
+    padding: 10px 20px;
+}
+
+nav ul li {
+    background: $menu-background;
+    position: relative;
+    float: left;
```

Comment:
hehe, will do! (I haven't done web dev in a decade so I don't know any of the new stuff yet!)